doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3222,6 +3222,13 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
+.. rst-class:: v2.6.0
+
+DRGN-21293: Le Read ISO TX Sync is implemented according to the raised errata ES-23138. 
+  In ES-23138, the return parameter TX_Time_Stamp is modified to be the SDU synchronization reference of the SDU previously scheduled for transmission, and not the CIG or BIG reference anchor point.
+  Note that there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference when the CIG or BIG is configured with an ISO_Interval that equals the SDU_Interval.
```

Comment:
```suggestion
  Note that when the CIG or BIG is configured with an ``ISO_Interval`` that equals the ``SDU_Interval``, there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference .
```
Alternatively, these could be just ISO interval and SDU interval, and not be parameters at all.